shop_stripe/payment.py

```diff
@@ -52,6 +57,31 @@ def charge(self, cart, request):
             msg = "Stripe returned status '{status}' for id: {id}"
             raise stripe.error.InvalidRequestError(msg.format(**charge))
 
+    def subscribe(self, cart, request):
+        """
+        Create a new customer and subscribe the customer
+        to the default payment plan.
+        """
+        user_id = cart.customer.user_id;
+        user = User.objects.get(id=user_id)
+
+        try:
+            customer = StripeCustomer.objects.get(user_id=user_id)
+            stripe_customer = stripe.Customer.retrieve(customer.stripe_customer_id)
+        except ObjectDoesNotExist:
```

Comment:
One suggestion here. This code is fine, but I would probably not use the generic ObjectDoesNotExist.  Each Django ORM model (StripeCustomer in this case? (This ambiguity will prove my point in a secont)) has it's own .DoesNotExist (double check I have the exception name right).  If you only expect to ever see StripeCustomer not existing, use it directly as in `except StripeCustomer.DoesNotExist`.  The reason is, you are more clear about what you're doing.

One other small issue, and I'm of two minds of this:  Some frown on using try/except as control blocks (if/else).  The code here reads fine, and actually is efficient in the sense of "ask for forgiveness instead of permission".  @amadornimbis can you weigh in?

Does stripe.Customer have something like a get_or_create()?  I think the Django ORM does, so you might have an opportunity there to streamline things.

I've tossed about 2.5 things up there, that are mostly only spatially related.  Let me know if I can be more clear.